src/superio/nuvoton/nct5104d/nct5104d.h

```diff
@@ -19,7 +19,6 @@
 #define SUPERIO_NUVOTON_NCT5104D_H
 
 /* SIO global configuration */
-#define LDN_SELECT_CR07     0x07
```

Comment:
@piotr-kleins I have actually thought of leaving the LDN_SELECT and remove GLOBAL_OPTION_CR7. WHy? Becase global registr 0x7 is always an ldn select in all superio's. It only confuses naming this register as global option

Reply:
@piotr-kleins you have introduced unnecessary change. Look at the space between LDN_SELECT_CR07 and 0x07. Please revert to original state